Community/Tdarr_Plugin_MC93_Migz1FFMPEG.js

```diff
@@ -335,8 +366,75 @@ const plugin = (file, librarySettings, inputs, otherArguments) => {
     response.preset = '-c:v vp8_cuvid';
   }
 
-  response.preset += `${genpts}, -map 0 -c:v hevc_nvenc -cq:v 19 ${bitrateSettings} `
-  + `-spatial_aq:v 1 -rc-lookahead:v 32 -c:a copy -c:s copy -max_muxing_queue_size 9999 ${extraArguments}`;
+  // Determine if GPU is present and which gpu to use based on utilization of each GPU
+  response.infoLog += 'Running nvidia-smi to see if a GPU is available and selecting the GPU with less utilization\n';
+  let gpu_num = -1;
+  let gpu_util = 100000;
+  let result_util = 0;
+  let gpu_count = '';
+  let gpu_res = '';
```

Comment:
gpu_res is an array, initialize it like one

Reply:
the first command `gpu_res = execSync('nvidia-smi --query-gpu=name --format=csv,noheader');` outputs a string with all available GPU names each in a new line (or an error if nvidia-smi is not installed), which will then be converted to an array. :)